hubspot/plugin.py

```diff
@@ -2317,3 +2343,83 @@ def hubspot_fetch_associated_object_ids(
             ]
 
     return []
+
+
+ASSOCIATION_TYPE_IDS = {
+    "CONTACT_TO_CONTACT": 449,
+    "CONTACT_TO_COMPANY": 279,
+    "CONTACT_TO_PRIMARY_COMPANY": 1,
+    "CONTACT_TO_DEAL": 4,
+    "COMPANY_TO_COMPANY": 450,
+    "COMPANY_TO_CONTACT": 280,
+    "COMPANY_TO_DEAL": 342,
+}
+
+
+@dataclass
+class HubSpotAssociationType:
+    type: Literal[
+        "CONTACT_TO_CONTACT",
+        "CONTACT_TO_COMPANY",
+        "CONTACT_TO_PRIMARY_COMPANY",
+        "CONTACT_TO_DEAL",
+        "COMPANY_TO_COMPANY",
+        "COMPANY_TO_CONTACT",
+        "COMPANY_TO_DEAL",
+    ]
+
+
+def hubspot_create_association_between_object_ids(
+    association_type: HubSpotAssociationType,
+    source_object_type: Union[HubSpotObjectType, HubSpotCustomObjectType],
+    source_object_id: str,
+    target_object_type: Union[HubSpotObjectType, HubSpotCustomObjectType],
+    target_object_id: str,
+) -> List[str]:
+    """
+    Returns the IDs of target objects associated with the source object
+    using the HubSpot association API. You must use this to find HubSpot
```

Comment:
find -> create

also what does this return?

should it just returned number of associations created?

Reply:
Updated doc.